src/storage/user-data.ts

```diff
@@ -107,7 +107,7 @@ export async function saveUserData( data: UserData ): Promise< void > {
 	await writeFile( filePath, asString, 'utf-8' );
 }
 
-const LOCKFILE_PATH = nodePath.join( getResourcesPath(), LOCKFILE_NAME );
+const LOCKFILE_PATH = getUserDataLockFilePath();
```

Comment:
I'm unsure why in https://github.com/Automattic/studio/pull/1319 we used two different locations for the app lock file. 🤔

Reply:
It's not only an improvement, but a fix for one of the bugs reported in STU-612. @ndiego reported layout issues on crash screen, but the crash in steps he shared shouldn't occur in the first place. The lock file location fixes that crash.

It would make sense to have two separate issues, but I addressed both as part of one PR, as both problems were reported under one issue and are related - one reproduces the other.